package.json

```diff
@@ -20,12 +16,17 @@
     "grunt-shell": "1.3.0",
     "shelljs": "0.3.0"
   },
+  "scripts": {
+    "start": "electron main.js",
+    "test": "electron tests.js",
+    "postinstall": "cd ./node_modules/leveldown && node-gyp rebuild --target=1.4.1 --dist-url=https://atom.io/download/atom-shell"
+  },
   "dependencies": {
     "electron": "1.4.1",
     "request": "2.69.0",
-    "infusion": "2.0.0-dev.20161027T155641Z.0bcd507",
-    "windows": "git://github.com/GPII/windows.git#24cdf9164d0696a1617db250573e205b4568385b",
-    "universal": "git://github.com/GPII/universal.git#b58d1859e4d53303a1fdc4d74640e0c43af345e5",
+    "infusion": "git://github.com/amb26/infusion.git#240c00be023732ac8dcc193a57bc1b46c58d2de7",
```

Comment:
This is a copy of the dependency from universal (the one referenced from the windows repo). @amb26 do you know what the state is of dependencies on your infusion branch?